lib/ao_host.js

```diff
@@ -112,13 +112,6 @@ class AOHost extends AsyncEventEmitter {
 
     this.adapter.once('order:snapshot', (snapshot) => {
       this.orderSnapshot = snapshot
-      this.loadAllAOs().then(() => { // eslint-disable-line
-        this.emit('ready')
```

Comment:
shouldn't it still emit `ready` so other code knows that algo orders are set up?